### DISCUSSION TITLE
Release/5.9.3

### DISCUSSION THREAD

**Discussion by Rixxan** [EDCD/EDMarketConnector#2052] [Announcement]
Release 5.9.3
===
This release is identical to 5.9.3, except reverts a bad change. 

- REVERTS Deprecated load_module() is now retired (#1462)

<hr /><em>This discussion was created from the release <a href='https://github.com/EDCD/EDMarketConnector/releases/tag/Release/5.9.3'>Release/5.9.3</a>.</em>

**Comment by Kallum-Maari** [comment]
Hi there, sorry to bother you. I'm having issues with EDMC repetitively asking to be updated to 5.9.3. When it finishes, it persistently loads the frontier login screen but doesn't actually connect, just seemingly runs in a loop as if requests were unanswered or some such. I've tried a full uninstall and reinstall and it still hasn't resolved it. (yes i turned it off and back on too)

**Reply by Rixxan** [reply]
Hey there! Thanks for reporting. If you can, in your install directory (by default, `C:\Program Files (x86)\EDMarketConnector`) try running the `EDMarketConnector - localserver-auth.bat` file?

That sometimes helps with FDEV authentication issues. Some other resources that might help are here: 

https://github.com/EDCD/EDMarketConnector/wiki/Troubleshooting#problems-with-frontier-authentication

**Reply by Kallum-Maari** [reply]
Sorry that I didn't think to look for that page first. You're brilliant, thank you so much for your help, it worked flawlessly.
